Dockerfile

```diff
@@ -96,6 +96,9 @@ ENV CASSANDRA_CLUSTER=false \
 # Configure search nodes. Empty string falls back to search being done on the nodes themselves.
 ENV PEGA_SEARCH_URL=
 
+# Configure requestor passivation timeout
+ENV WEB_REQUESTOR_PASSIVATION_TIMEOUT=3600
```

Comment:
Does this need to be picked up by Pega?  If so, it may need to be added to [setenv.sh](https://github.com/pegasystems/docker-pega-web-ready/blob/master/tomcat-bin/setenv.sh) as well.

Also, should this default to a lower number, like 600? (10min instead of 60min)

Since this is just changing requestor passivation timeout behind the scenes, maybe we should drop WEB_ from the name and just call it REQUESTOR_PASSIVATION_TIMEOUT?

Reply:
The question is do I need to define this variable? The intent is to set it via helm charts and for this probably I don't need to define it here, right?
This probably would be needed if someone deploys Pega without using helm charts and want to set requestor timeout during deployment.

Reply:
You probably don't - thats a good point.  Since this configuration is effectively moot without an appropriate load balancer config too, maybe it should just be in the helm chart.

If the current image works without modification, feel free to close this PR.

Reply:
Agree, change not needed.